### ISSUE TITLE
Install on Void Linux

### ISSUE THREAD

**Issue by robotamer** [vlang/v#17139]
### Describe the bug

`# xbps-install base-devel
`
____

```
$ git clone https://github.com/vlang/v
$ cd v

```
____
```
$ make 

cd ./vc && git clean -xf && git pull --quiet
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=gnu99 -w -o v1.exe ./vc/v.c -lm -lpthread 
./v1.exe -no-parallel -o v2.exe  cmd/v
==================
tcc: error: library 'atomic' not found
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.

This is a compiler bug, please report it using `v bug file.v`.

https://github.com/vlang/v/issues/new/choose

You can also use #help on Discord: https://discord.gg/vlang

make: *** [GNUmakefile:114: all] Error 1
```

### Expected Behavior

install

### Current Behavior

tcc: error: library 'atomic' not found

### Reproduction Steps

make

### V version

V 0.3.2 d3e4058

### Environment details (OS name and version, etc.)

**Void Linux musl**

```
$ uname -r
6.1.6_1

```
_______

```
$ ./v1.exe -v
V 0.3.2 d3e4058
```

**Comment by MatejMagat305** [comment]
I'm not expert, but you could try install: `xbps-install -Su libatomic_ops-devel`

**Comment by robotamer** [comment]
That did not work.
I am getting the same error:
`tcc: error: library 'atomic' not found`

Any more ideas?
Thank you

```
# xbps-install -Su libatomic_ops-devel
[*] Updating repository `https://repo-default.voidlinux.org/current/musl/x86_64-musl-repodata' ...
[*] Updating repository `https://repo-default.voidlinux.org/current/musl/nonfree/x86_64-musl-repodata' ...

Name                Action    Version           New version            Download size
libatomic_ops       install   -                 7.6.14_1               23KB 
libatomic_ops-devel install   -                 7.6.14_1               69KB 

```

**Comment by imiric** [comment]
`libatomic_ops` doesn't seem to be the correct package, but `xbps-install -S libatomic-devel` worked for me on Void Linux.

**Comment by robotamer** [comment]
@imiric Thanks that worked.
Here is all you need:
```
# xbps-install -Su base-devel
# xbps-install libatomic-devel
$ git clone https://github.com/vlang/v
$ cd v
$ make
```

**Comment by JalonSolov** [comment]
Actually, this wouldn't be a bad thing to have documented.  There are docs now for various other distros.  Would you be interested in submitting a PR?

**Comment by robotamer** [comment]
@JalonSolov 
 Void Linux instructions pull request #17176

**Comment by spytheman** [comment]
Thanks for your PR.
Fixed by 9809427 .
